heat/core/statistics.py

```diff
@@ -1535,10 +1537,24 @@ def std(
     >>> ht.std(a, 1)
     DNDarray([1.2961, 0.3362, 1.0739, 0.9820], dtype=ht.float32, device=cpu:0, split=None)
     """
-    if not axis:
-        return np.sqrt(var(x, axis, ddof, **kwargs))
+    if not isinstance(ddof, int):
+        raise TypeError(f"ddof must be integer, is {type(ddof)}")
+    elif ddof > 1:
+        raise NotImplementedError("Not implemented for ddof > 1.")
```

Comment:
this is "only" not implemented for the distributed or GPU case, correct?

Reply:
i think it wasnt allowed by torch when i wrote that. but i guess it is now. ill take that out. worst case, we put it back in later